script/src/syscalls/mod.rs

```diff
@@ -31,9 +31,9 @@ pub const ITEM_MISSING: u8 = 2;
 pub const SLICE_OUT_OF_BOUND: u8 = 3;
 
 pub const LOAD_TRANSACTION_SYSCALL_NUMBER: u64 = 2051;
+pub const LOAD_SCRIPT_SYSCALL_NUMBER: u64 = 2052;
```

Comment:
Why change the syscall number? cc @xxuejie

Reply:
To keep syscall number consistent, the number `205x` is used to describe load the whole struct, and `206x` is used to describe load the hash of the struct.